_observing-your-data/query-insights/query-insights-dashboard.md

```diff
@@ -182,6 +183,86 @@ When configuring the query insights feature, remember the following best practic
+
+| Panel  | Description  |
+| :--- | :--- |
+| **Active queries**       | The total number of currently executing queries.                                |
+| **Avg. elapsed time**    | The average execution time of live queries.                                     |
+| **Longest running query**| The duration and ID of the longest currently executing query.                   |
+| **Total CPU time**       | The cumulative CPU time used by all active queries.                             |
+| **Total memory usage**   | The total memory consumed by live queries.                                      |
+
+### Breakdown charts
+
+Two visual charts provide breakdowns of query load:
+
+- **By node** – Shows how many queries are running on each node.
+- **By index** – Displays how many queries are targeting each index.
+
+![Live Queries visual charts]({{site.url}}{{site.baseurl}}/images/Query-Insights/Live_queries_visuailization.png)
+
+You can toggle between **Donut** and **Bar** chart formats using the chart type switch.
+
+Only the top 9 items are displayed individually in the chart; additional values are grouped under the **Others** category.
+
+### Live query table
+
+The live query table lists the following information for each live query.
+
+![Live Queries Table]({{site.url}}{{site.baseurl}}/images/Query-Insights/Live_Queries_Table.png)
+
+| Column  | Description |
+| :--- | :--- |
+| **Timestamp**        | The time when the query started running.                            |
+| **Task ID**          | The unique identifier for the search task.                            |
+| **Index**            | The index or indexes the query is targeting.                          |
+| **Node**             | The node currently running the query.                               |
+| **Time elapsed**     | The execution time for the query so far (formatted).                                |
+| **CPU usage**        | The cumulative CPU time used by the query.                            |
+| **Memory usage**     | The amount of memory consumed by the query so far.                              |
+| **Search type**      | The search execution method, such as `query_then_fetch`.               |
+| **Coordinator node** | The node that coordinated the query.                                  |
+| **Status**           | The status of the query task. Can be either `running` or `cancelled`. |
+| **Actions**          | The available controls, such as canceling the query.                  |
+
+You can use the filter bar to search for queries by text or specific field values---such as node ID, index name, or task ID---and paginate the table to better analyze specific queries. The following image shows the live queries table view.
+
+![Live Queries Table]({{site.url}}{{site.baseurl}}/images/Query-Insights/Live_Queries_Table.png)
+
+The **Live queries** table provides the following real-time monitoring controls:
```

Comment:
Should we be using "**Live queries**" or "live queries". Please make consistent throughout.